openmls/src/group/mls_group/test_mls_group.rs

```diff
@@ -1316,6 +1316,169 @@ fn builder_pattern(ciphersuite: Ciphersuite, provider: &impl OpenMlsProvider) {
     assert_eq!(builder_err, InvalidExtensionError::IllegalInLeafNodes);
 }
 
+// Test the successful update of Group Context Extension with type Extension::Unknown(0xff11)
+#[apply(ciphersuites_and_providers)]
+fn update_group_context_with_unknown_extension(
+    ciphersuite: Ciphersuite,
+    provider: &impl OpenMlsProvider,
+) {
+    let (alice_credential_with_key, _alice_kpb, alice_signer, _alice_pk) =
+        setup_client("Alice", ciphersuite, provider);
+
+    // === Define the unknown group context extension and initial data ===
+    let unknown_extension_data = vec![1, 2];
+    let unknown_gc_extension = Extension::Unknown(0xff11, UnknownExtension(unknown_extension_data));
```

Comment:
It's just a test, but it would be good to store `0xff11` in a variable and use that throughout the test instead of the literal number.

Reply:
👍 updated 7172d638294fb6c65deb65eaf6ce6045820de982